ocaml/xapi/xapi_pgpu.ml

```diff
@@ -312,6 +312,17 @@ let update_dom0_access ~__context ~self ~action =
 	| `disabled,          `disable
 	| `enable_on_reboot,  `disable -> `disabled
 	in
+
+	let rc = Db.PGPU.get_record ~__context ~self in
+	let pci = rc.API.pGPU_PCI in
```

Comment:
These two lines can be replaced with

```
let pci = Db.PGPU.get_PCI ~__context ~self in
```

There's no need to get the whole record if we only want one field from it.